guides/security-credential-store-enc-exp.adoc

```diff
@@ -0,0 +1,163 @@
+= Using Credential Stores With Encrypted Expressions on WildFly
```

Comment:
This should now be updated to "with WildFly", same with the title in security-credential-store-for-passwords.adoc

Reply:
fixed!